pkg/handler/repository_parameters_test.go

```diff
@@ -16,16 +16,32 @@ import (
 	"github.com/openlyinc/pointy"
 	"github.com/redhatinsights/platform-go-middlewares/identity"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/suite"
 )
 
-func serveRepositoryParametersRouter(req *http.Request, mockDao *MockRepositoryConfigDao) (int, []byte, error) {
+type MockExternalResourceDao struct {
+	mock.Mock
+}
+
+func (erd *MockExternalResourceDao) ValidRepoMD(url string) (int, error) {
+	args := erd.Called(url)
+	return args.Int(0), args.Error(1)
+}
+
+func (erd *MockExternalResourceDao) FetchGpgKey(url string) (string, error) {
+	args := erd.Called(url)
+	return args.String(), args.Error(1)
+}
+
```

Comment:
hrmm, we already have a mockExternalResource in pkg/dao/repository_config_test.go   I wonder if we should just move it to a new package like  'test' ?

Reply:
Sure! can you tell me exactly where to move it from "xxx" to "xxx/xxx". 
Most of this structure still doesn't make a lot of sense to me, despite starting to see the pattern in the layout.

Reply:
so right now there is a mockExternalResource in pkg/dao/repository_config_test.go and now you've added a duplicate in pkg/handler/repository_parameters_test.go.  lets just have one and put it somewhere like:

.pkg/tests/mocks.go  ?

Reply:
and i could see us moving all the mock implementation there, so maybe it should be ./pkg/test/mocks/external_resources.go ?

Reply:
I like it. Will do.